src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -3917,21 +3917,44 @@ s32 Player_TryActionChangeList(PlayState* play, Player* this, s8* actionChangeLi
     return false;
 }
 
-s32 func_808374A0(PlayState* play, Player* this, SkelAnime* skelAnime, f32 arg3) {
+typedef enum {
+    /* -1 */ PLAYER_INTERRUPT_NONE = -1,
+    /*  0 */ PLAYER_INTERRUPT_NEW_ACTION,
+    /*  1 */ PLAYER_INTERRUPT_MOVE
+} PlayerActionInterruptResult;
```

Comment:
can you double-check retail bss ordering?

Reply:
![image](https://github.com/zeldaret/oot/assets/22045047/d987102f-8a7f-4f97-a250-c8a8f2a9cc5f)